CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +15,20 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
-        //Methods
+        Die die1 = new Die();
+        Die die2 = new Die();
+        Die die3 = new Die();
 
+        public (int, int, int, int) StartGame() //Method
+        {
+            int roll1 = die1.Roll();
+            int roll2 = die2.Roll();
+            int roll3 = die3.Roll();
+
+            int sum = roll1 + roll2 + roll3;
+            Console.WriteLine($"Die 1 = {roll1} \nDie 2 = {roll2} \nDie 3 = {roll3}");
+
+            return (sum, roll1, roll2, roll3);
+        }
```

Comment:
Dice rolls could be done continuously by utilising condition-controlled iteration.

Reply:
I agree that repeat roll functionality would be a great addition to the code.